### PR TITLE
FIlter language tabs on node page to show only languages used in a SiteAlias

### PR DESCRIPTION
**Pull request by itkg-ngilain**
[OO-FEATURE] FIlter language tabs on node page to show only languages used in a SiteAlias

### CODE REVIEW

File: ApiBundle/Controller/SiteController.php [hunk 1]
```diff
@@ -93,4 +93,25 @@ public function deleteAction($siteId)
...
+     *
+     * @return Response
+     */
+    public function getAllLanguagesAction($siteId)
+    {
+        $site = $this->get('open_orchestra_model.repository.site')->findOneBySiteId($siteId);
+
+        $availableLanguages = array();
+
+        foreach ($site->getAliases() as $alias) {
```
Comment (alavieille) [inline ApiBundle/Controller/SiteController.php]: why not used getLanguages ?

Comment (itkg-ngilain) [inline ApiBundle/Controller/SiteController.php]: Only because I didn't notice these method :) I update the PR

**Comment by itkg-ngilain** [comment]
@alavieille : PR updated

**Comment by itkg-nanne** [comment]
:+1:

**Pull request merged**